postBuild

```diff
@@ -1,21 +1,11 @@
 #!/bin/bash
 
-CODE_SERVER_VERSION="2.1692-vsc1.39.2"
-
 # Enable the proxy extension in notebook and lab
 jupyter serverextension enable --py jupyter_server_proxy
 jupyter labextension install jupyterlab-server-proxy
 jupyter lab build
 
-# Download and install VS Code server
-wget -q -O code-server.tar.gz https://github.com/cdr/code-server/releases/download/${CODE_SERVER_VERSION}/code-server${CODE_SERVER_VERSION}-linux-x86_64.tar.gz
-mkdir -p ~/.local/bin
-tar -xvf code-server.tar.gz -C ~/.local/bin --strip-components=1
-rm -rf code-server.tar.gz
-
-wget https://github.com/microsoft/vscode-python/releases/download/2019.10.44104/ms-python-release.vsix
-code-server --install-extension ms-python-release.vsix
-rm -rf ms-python-release.vsix
+code-server --install-extension ms-python.python
```

Comment:
Did the name of the extension change? (why did we go from `ms-python-release.vsix` to `ms-python.python`?)

Reply:
No, these are different names.

- `vscode-python` is the name of the extension
- `ms-python-release.vsix` is the name of the *release* tarball on Github
- `ms-python.python` is the id in the code-server extension marketplace.